packages/nodes-base/package.json

```diff
@@ -830,6 +830,10 @@
       "dist/nodes/Transform/Summarize/Summarize.node.js"
     ]
   },
+  "exports": {
```

Comment:
I think we should extract out these `exports` and all the `/dist` removals into a separate PR, since those will work tsc and node as well.